react/ContactPicker/index.jsx

```diff
@@ -1,7 +1,8 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 import cx from 'classnames'
-import { Contact } from 'cozy-doctypes'
+import { models } from 'cozy-client'
+const { getDisplayName } = models.contact
```

Comment:
nit: it would be less surprising to define getDisplayName after last import line 8